docs/UserGuide.md

```diff
@@ -176,34 +201,47 @@ to delete all the sample entries and start using your very own ClientBook.
 
 | Feature | Description |
 | ----------- | ----------- |
-| [`help`](#help-viewing-help) | Viewing help |
+|<span style="color:#b573c9">**Contact Management**</span>|
 | [`add`](#add-add-client-contact) | Add client contact |
 | [`edit`](#edit-edit-client-contact) | Edit client contact |
+| [`delete`](#delete-delete-client-contact) | Delete client |
+| [`batch`](#batch-execute-commands-in-batch) | Execute commands in batch |
+| [`meet`](#meet-schedule-a-meeting-with-a-client) | Schedule a meeting with a client |
+|<span style="color:#c93640">**Contact Viewing**</span>|
 | [`list`](#list-list-all-clients) | List all clients |
 | [`find`](#find-search-for-client-contact-based-on-keywords) | Search for client contact based on keywords |
 | [`policy`](#policy-display-policies-associated-with-a-client) | Display policies associated with a selected client |
-| [`delete`](#delete-delete-client-contact) | Delete client |
-| [`batch`](#batch-execute-commands-in-batch) | Execute commands in batch |
 | [`sort`](#sort-sort-list-of-clients) | Sort list of clients |
-| [`meet`](#meet-schedule-a-meeting-with-a-client) | Schedule a meeting with a client |
+|<span style="color:#c9a500">**Shortcuts**</span>|
 | [`addshortcut`](#addshortcut-add-shortcut) | Add shortcut |
 | [`editshortcut`](#editshortcut-edit-shortcut) | Edit shortcut |
 | [`deleteshortcut`](#deleteshortcut-delete-shortcut) | Delete shortcut |
 | [`listshortcut`](#listshortcut-list-all-shortcuts) | List all shortcuts |
-| [`clearshortcut`](#clearshortcut-clear-all-shortcuts) | Clear all shortcuts |
+| [`clearshortcut`](#clearshortcut-delete-all-shortcuts) | Clear all shortcuts |
+|<span style="color:#3cb44b">**General**</span>|
 | [`lock`](#lock-lock-clientbook-with-a-user-selected-password) | Lock ClientBook with a user-selected password |
 | [`unlock`](#unlock-unlock-clientbook) | Unlock ClientBook |
+| [`help`](#help-viewing-help) | Viewing help |
 | [`exit`](#exit-exiting-the-program) | Exit ClientBook |
+| [Keyboard commands](#keyboard-commands) | Keyboard commands to enter command keywords quickly |
 | [Saving data](#saving-data) | Save the data to a file in the home folder |
 | [Editing data file](#editing-data-file) | Edit the data stored in the file in the home folder|
 
 ### What information can we store for each client contact?
 
+
+Each client attribute such as name and address have been assigned identifiers so that you do not have to type out
+the entire attribute when using ClientBook. For example, name has been abbreviated to n. Listed below are all the identifiers
```

Comment:
Do you think it might be clearer if n is mentioned in quotes?

Reply:
No, theres no special reason to use markup to highlight n as it is not a command or anything that user needs to enter.